include/pybind11/cast.h

```diff
@@ -469,6 +469,7 @@ template <typename type> class type_caster<std::reference_wrapper<type>> : publi
     public: \
         static PYBIND11_DESCR name() { return type_descr(py_name); } \
         static handle cast(const type *src, return_value_policy policy, handle parent) { \
+            if (!src) return none().release(); \
```

Comment:
I'm not familiar with the pybind code, but in other places in this file None is returned with `return none().inc_ref();` rather than `.release()`?